pkg/version/check.go

```diff
@@ -75,9 +74,9 @@ func removeVAndHash(v string) string {
 }
 
 // CheckClusterVersion check TiKV and PD version.
+// need only one PD alive and match the cdc version.
 func CheckClusterVersion(
-	ctx context.Context, client pd.Client, pdHTTP string, credential *security.Credential, errorTiKVIncompat bool,
-) error {
+	ctx context.Context, client pd.Client, pdAddrs []string, credential *security.Credential, errorTiKVIncompat bool) error {
```

Comment:
Only one pd is needed to be valid, but we pass in all the addresses, which is a bit strange for me. But I think it's acceptable.

What do you think?

Reply:
Because we need to find a suitable one from all PDs, we can only pass all PDs in.
Do you have a better way? If so, please tell me, thanks.